src/pkg/backup/details/sharepoint.go

```diff
@@ -58,7 +61,7 @@ func (i SharePointInfo) Headers() []string {
 	case SharePointLibrary:
 		return []string{"ItemName", "Library", "ParentPath", "Size", "Owner", "Created", "Modified"}
 	case SharePointList:
-		return []string{"ListName", "ListItemsCount", "Owner", "Created", "Modified"}
+		return []string{"ListName", "ListItemsCount", "SiteURL", "Template", "Owner", "Created", "Modified"}
```

Comment:
There's a couple conflicts between this and our standard display:
* we don't normally show the SiteURL, since that's already identified by the backup resource.
* `Owner` is shown here, but not populated as part of the item info.
* These names should be very human readable.  Prefer words like `List` over `ListName`, and `Items` over `ListItemsCount`.

Reply:
I will drop the `Owner` and make the changes the column names.

Reply:
> I thought we could use SiteURL as a selector and hence it would be useful for it to be displayed

Ah, good question.  That has two use cases.
1. when showing `sharepoint` app details, each backup is unique to the resource owner- IE: the site.  Therefore, displaying the site URL is redundant, because you cannot select any other site.
2. when showing `groups` app details, since each backup may contain multiple sites (because the group is the resource, and groups many contain many sites), we might want to show the site URL in that case.

Also, we probably want to name the column `Site`, instead of `SiteURL`.